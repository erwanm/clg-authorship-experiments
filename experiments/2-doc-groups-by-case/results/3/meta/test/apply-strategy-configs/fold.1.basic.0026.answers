1916-ew-k-sh:1907-ew-tfott:1897-ew-tdof 1919-ew-m	0.199158
1883-cfw-ftm:1889-cfw-jl:1875-cfw-cn 1899-hj-taa	0.788376
1918-us-jh:1908-us-tmc:1913-us-dg 1878-cdw-hikab	0.42817
1917-hj-tit:1898-hj-ttm:1886-hj-tb 1864-lma-m	0.30959
1877-lma-amm:1872-lma-ss:1864-lma-opdaot 1864-lma-m	0.123371
1915-rwc-b:1912-rwc-tsoa:1918-rwc-tlg 1904-rwc-isotu	0.260253
1874-mt-lotm:1906-mt-cfma:1916-mt-tms 1909-mt-isd	1
1914-us-sm:1922-us-tcmc:1903-us-tjoas 1887-espw-jtf	0.491353
1876-cdw-itl:1891-cdw-oi:1889-cdw-sths 1894-cdw-tgh	0.76131
1864-lma-m:1867-lma-tagomtt:1882-lma-ajsb-v6 1868-lma-lw	1
1866-wta-bos:1895-wta-alae:1876-wta-vao 1901-fmc-m	0.622956
1898-fmc-vc:1885-fmc-tcotk:1895-fmc-t 1903-us-og	0.395177
1893-fmc-md:1885-fmc-atoalp:1898-fmc-ari 1884-fmc-aap	0.774943
1889-es-atg:1905-es-tpoe:1888-es-ttatv 1913-es-tm	0.591512
1911-rwc-tgr:1910-rwc-ap:1922-rwc-tfj 1871-hbs-ofs	0.42525
1911-hj-to:1880-hj-ws:1878-hj-aie 1903-ew-ivatg	0
1904-espw-t:1866-espw-gcj:1866-espw-gb 1891-cdw-oi	0.308957
1903-us-ach:1908-us-tmc:1903-us-tjoas 1914-us-sm	0.582915
1882-lma-ajsb-v6:1877-lma-amm:1864-lma-opdaot 1909-rwc-tdm	0.271208
1910-rwc-ap:1900-rwc-taoamm:1906-rwc-ttolp 1909-rwc-tdm	0.514404
1897-cdw-tpfwsw:1891-cdw-oi:1878-cdw-hikab 1916-ab-tp	0.121855
1922-rwc-tfj:1900-rwc-taoamm:1911-rwc-tcl 1866-espw-gsar	0.48984
1895-cfw-mcac:1875-cfw-cn:1883-cfw-ftm 1886-hbs-tsw	0.735318
1868-wta-mob:1863-wta-isao:1884-wta-Sac 1893-espw-dm	0.59213
1910-us-sts:1911-us-alp:1908-us-tmc 1903-us-og	0.649805
1885-es-tpod:1913-es-tm:1905-es-tpoe 1889-es-atg	1
1897-ew-tdof:1919-ew-m:1913-ew-tcotc 1916-ew-k-sh	0.227441
1916-ab-tp:1899-ab-tt:1895-ab-mgtonel 1884-ab-sbts	0.76453
1871-espw-tsp:1901-espw-a:1866-espw-gcj 1904-espw-t	0
1922-us-tcmc:1917-us-kc:1906-us-tj 1918-us-jh	0.579762
1885-es-tpod:1889-es-tptk:1917-es-ow-sh 1905-es-tpoe	0
1909-mt-isd:1869-mt-ia:1902-mt-adbds 1896-mt-projoa	0.551262
1903-ew-ivatg:1907-ew-tfott:1913-ew-tcotc 1874-wta-tcw	0.413984
1874-cfw-tac:1875-cfw-cn:1889-cfw-jl 1911-rwc-tgr	0.272013
1901-fmc-m:1895-fmc-t:1891-fmc-do 1885-fmc-atoalp	0.717574
1896-ab-boat:1922-ab-oc:1895-ab-mgtonel 1922-rwc-tfj	0.330422
1894-cdw-tgh:1891-cdw-oi:1889-cdw-sths 1878-cdw-hikab	1
1896-ab-boat:1899-ab-tt:1884-ab-sbts 1907-ab-rm	1
1887-fmc-s:1884-fmc-aap:1885-fmc-zamc 1868-wta-mob	0.887648
1889-es-tptk:1888-es-ttatv:1907-es-tlotg 1905-es-tpoe	0
1922-us-tcmc:1917-us-kc:1901-us-km 1903-us-tjoas	0.38772
1917-es-ow-sh:1889-es-tptk:1907-es-tlotg 1868-lma-lw	0.259597
1910-rwc-ap:1911-rwc-tgr:1912-rwc-tsoa 1913-rwc-thc	0
1887-espw-tgb:1896-espw-cfal:1871-espw-tsp 1900-espw-wtg	0
1893-fmc-md:1907-fmc-tdr:1883-fmc-tl 1917-us-kc	0.74315
1894-espw-asl:1866-espw-gcj:1869-espw-mwag 1912-rwc-tsoa	0.177707
1879-haj-ttb:1906-haj-rotr-ph:1866-haj-tcw 1871-haj-tt	0.565256
1880-hj-ws:1886-hj-pc:1896-hj-toh 1921-us-tbol	0.547626
1874-cfw-tac:1884-cfw-ea:1875-cfw-cn 1872-wta-lb	0.652159
1898-hj-ttm:1883-hj-rh:1885-hj-gr 1916-ab-tp	0.412182
1904-hj-tgb:1885-hj-gr:1883-hj-rh-v2 1868-wta-dad	0.535515
1891-cdw-oi:1897-cdw-tpfwsw:1894-cdw-tgh 1885-fmc-tcotk	0.275825
1883-wta-sh:1876-wta-vao:1864-wta-tsb 1891-wta-sbtu	1
1863-wta-isao:1855-wta-aa:1881-wta-utr 1858-wta-pap	0.767936
1869-mt-ia:1881-mt-tpatp:1884-mt-taohf 1916-mt-tms	0.155606
1889-espw-tsfi:1867-espw-gyatgc:1866-espw-gcj 1914-mh+-mhce-nf	0.67575
1891-wta-sbtu:1858-wta-pap:1889-wta-wtel 1874-wta-tcw	0.540977
1863-wta-isao:1882-wta-aa:1880-wta-ds 1855-hbs-tmf	1
1867-lma-tagomtt:1864-lma-opdaot:1877-lma-amm 1903-ew-ivatg	0.296014
1895-cfw-mcac:1875-cfw-cn:1883-cfw-ftm 1868-haj-rd	0.668789
1871-hbs-mwai:1859-hbs-tmw:1854-hbs-smofl 1869-mt-ia	0.834657
1886-hj-pc:1909-hj-ih:1898-hj-itc 1878-hj-te	0.632475
1895-fmc-ajs:1885-fmc-atoalp:1893-fmc-kl 1906-fmc-a	1
1913-rwc-thc:1897-rwc-l:1911-rwc-tcl 1910-rwc-ap	0.547659
1871-hbs-mwai:1886-hbs-tsw:1855-hbs-tmf 1871-hbs-ofs	0.621758
1916-mt-tms:1881-mt-tpatp:1884-mt-taohf 1874-mt-lotm	0.725617
1883-espw-btg:1895-espw-tsojc:1887-espw-jtf 1889-wta-wtel	0.846036
1917-es-ow-sh:1885-es-tpod:1888-es-ttatv 1913-es-tm	0.069595
1909-hj-ih:1903-hj-ta:1885-hj-gr 1878-hj-te	0.782861
1921-ab-lig:1917-ab-bn:1910-ab-jwf 1922-ab-oc	0.643592
1903-us-tjoas:1906-us-tj:1922-us-tcmc 1911-us-alp	0.964116
1904-rwc-sm:1900-rwc-taoamm:1919-rwc-tsos 1910-us-sts	0.120722
1884-mt-taohf:1909-mt-isd:1902-mt-adbds 1897-ewaoc-tdof	0
1906-us-tj:1903-us-tjoas:1910-us-sts 1913-us-dg	0.194578
1881-mt-tpatp:1872-mt-ri:1916-mt-tms 1884-mt-taohf	0.444402
1882-wta-aa:1885-wta-sts:1876-wta-vao 1921-us-tbol	0.229807
1885-lma-ll:1882-lma-ajsb-v6:1877-lma-amm 1921-ab-lig	0.603525
1907-es-tlotg:1888-es-ttatv:1885-es-tpod 1873-hbs-wish	0.625403
1897-ab-tdohy:1910-ab-cn:1917-ab-bn 1920-us-1tsoap	0.14823
1899-hj-taa:1881-hj-tpoal:1898-hj-ttm 1895-cfw-mcac	0.650656
1871-hbs-mwai:1878-hbs-pp:1855-hbs-tmf 1876-cdw-itl	0.488222
1884-wta-Sac:1857-wta-ta:1890-wta-otb 1866-wta-bos	0.248783
1918-us-jh:1906-us-tj:1908-us-tmc 1914-us-sm	0.596571
1913-ew-tcotc:1919-ew-m:1907-ew-tfott 1871-hbs-mwai	0.86512
1897-ab-tdohy:1884-ab-sbts:1910-ab-cn 1889-fmc-g	0.223862
1878-hbs-pp:1870-hbs-tbc:1853-hbs-aktutc 1889-cfw-jl	0.617988
1899-haj-aoatb:1868-haj-faf:1874-haj-rftr 1875-haj-tyo	0.89136
1888-hj-tap:1881-hj-tpoal:1885-hj-altof 1880-hj-ws	1
1874-cfw-tac:1883-cfw-ftm:1875-cfw-cn 1873-hbs-wish	0.539939
1872-mt-ri:1869-mt-ia:1881-mt-tpatp 1878-cdw-hikab	0.693016
1887-espw-tgb:1866-espw-gsar:1883-espw-btg 1895-ab-mgtonel	0.632722
1884-cfw-ea:1875-cfw-cn:1883-cfw-ftm 1889-cfw-jl	1
1866-wta-ob:1858-wta-pap:1885-wta-sts 1863-wta-isao	1
1896-mt-projoa:1906-mt-cfma:1881-mt-tpatp 1917-us-kc	0.317603
1876-haj-btlb:1879-haj-ttb:1866-haj-hf 1881-haj-fcbtp	0.844128
1868-wta-mob:1881-wta-utr:1880-wta-ds 1872-wta-lb	0.996294
1878-cdw-hikab:1876-cdw-itl:1891-cdw-oi 1897-ewaoc-tdof	0.963593
1896-hj-tsop:1911-hj-to:1903-hj-ta 1897-ewaoc-tdof	0
1874-mt-lotm:1872-mt-ri:1902-mt-adbds 1869-mt-ia	0.702489
1913-es-tm:1888-es-ttatv:1889-es-tptk 1898-hj-ttm	1
