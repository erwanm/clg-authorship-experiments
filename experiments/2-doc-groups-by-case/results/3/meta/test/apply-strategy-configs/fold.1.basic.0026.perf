0.367098	0.601800	0.610000
