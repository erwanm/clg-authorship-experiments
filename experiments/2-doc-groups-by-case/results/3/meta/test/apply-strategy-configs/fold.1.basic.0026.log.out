apply-config.sh: applying config '/tmp/expe2/3/process/selected-meta-configs/fold.1.0038.model/strategy-configs/fold.1.basic.0026' to cases './results/3/meta/test/apply-strategy-configs/only-cases.tmp.4G3LFKv3'; results written to './results/3/meta/test/apply-strategy-configs/fold.1.basic.0026'
apply-config.sh: done.
